docs/LIBRARIES.md

```diff
@@ -0,0 +1,25 @@
+# Recommended libraries for use within Grants Stack
+
+- data validation: https://github.com/colinhacks/zod
+- unit testing: https://vitest.dev/
+- client-side data fetching: https://swr.vercel.app/
+- server-side fetch: https://github.com/npm/make-fetch-happen
+- server-side sqlite3: https://github.com/WiseLibs/better-sqlite3
+- sql querying: https://kysely.dev/
+- server-side logging: https://github.com/pinojs/pino
+- git hook management: https://github.com/evilmartians/lefthook/
+- nodejs execution in dev: https://github.com/esbuild-kit/tsx
+- nextjs - https://github.com/vercel/next.js
+- react query - I prefer it over swr https://tanstack.com/query/latest/docs/react/comparison
```

Comment:
we're not using react query anywhere right? we already mention swr above, we can probably remove this item